car/index.js

```diff
@@ -14,26 +25,188 @@ const headers = {
 };
 
 const client = createClient({
-  url: 'https://api-2.chargetrip.io/graphql',
+  url: 'https://api.chargetrip.io/graphql',
   fetchOptions: {
     method: 'POST',
     headers,
   },
   exchanges: [...defaultExchanges],
 });
 
+// Keeps track of the active search keyword. This way we always fetch the right results when using pagination.
+let searchKeyword = '';
+
+// Keeps track of which page of cars we are currently on.
+let currentPage = 0;
+
+/**
+ * The function that handles all our GraphQL networking alongside it's parameters.
+ * @param {Object} - Object that manages the page, size and search to be send towards our request
```

Comment:
``` @param { page: number, size: number, search: string } - Object that manages the page, size and search to be send towards our request```

You can define the keys in the objects to be more specific about the input object. Can you do this as well for the other functions?

Reply:
I'm fine with it but then I'll probably have to create an interface / type for this one;

* @param { Object } cars - A list of cars coming from the Chargetrip API

Not sure whether that's an overkill for an example.

Reply:
Creating the actual TS interfaces is not necessary since the examples are not in TS.